### ISSUE TITLE
Add React to existing project

### ISSUE THREAD

**Issue by rvanbael** [aws-amplify/amplify-js#5063]
Hey, I have an iOS running app on AWS backend, managed through Amplify. I want to add an admin react website to manage easily the backend data. I will [add a group to my cognito for admin](https://aws.amazon.com/fr/blogs/mobile/amplify-cli-enables-creating-amazon-cognito-user-pool-groups-configuring-fine-grained-permissions-on-groups-and-adding-user-management-capabilities-to-applications/). I will add a dedicated API for this group. Looks good.

But I am struggling for the react part. There are tons of tuto on how to create a new amplify project on react. But I cannot find how to add react on an existing project. To be precise, the "amplify init" command generates, after a push, a "aws-exports.js" file. My project does not have any "aws-exports.js" file. And I do not want to "init" a new project.

Any idea how could I leverage my existing backend with a react front-end?

**Comment by ashika01** [comment]
@rvanbael  Have you tried copying your `aws-exports` from existing into your react project ?

I guess there is 2 scenarios. 1) sharing aws-exports to share same backend. 2) you can use `amplify pull` with existing app id. [doc](https://aws-amplify.github.io/docs/cli-toolchain/quickstart#amplify-pull)

**Comment by rvanbael** [comment]
Thank you @ashika01  for your help.
I cannot share aws-exports file, as it does not exist. For iOS project that I use, this is not an aws-exports file that amplify generate. I think the equivalent are files called "awsconfiguration.json" and/or "amplifyconfiguration.json" that are in my iOS folder.

Same answer for amplify pull, there is no aws-exports file to pull.

I need to generate this file based on existing backend.

**Comment by ashika01** [comment]
@rvanbael Let me check that out and get back to you.

**Comment by ashika01** [comment]
@rvanbael Apologies. Completely got sidetracked. Did you figure this out by yourself? Could u see if you could created `aws-exports.js` out of the values in `awsconfiguration.json` file and check if that works?

**Comment by rvanbael** [comment]
Hello @ashika01, thanks for your response. No I am still at the same place, I switched to the front-end, waiting for your response. Yes it is what I had in mind if I had no response, trying to manually create the aws-exports.js file. But if you could save me this pain, and include this use case to roadmap, it would be great! I think it is not weird to have a project on both mobile & web with same backend... isn't it?

**Comment by ashika01** [comment]
@rvanbael: Yeah, I just wanted to check where you are at. This looks more like issue around CLI or maybe feature request. I will transfer this issue to the CLI repo in wither case.
